test/Atmos/EDMF/edmf_model.jl

```diff
@@ -0,0 +1,284 @@
+#### EDMF model
+
+#### Entrainment-Detrainment model
+
+"""
+    EntrainmentDetrainment
+An `EntrainmentDetrainment` model for EDMF, containing
+all related free parameters.
+"""
+Base.@kwdef struct EntrainmentDetrainment{FT <: AbstractFloat}
+    "Entrainmnet TKE scale"
+    c_λ::FT = 0.3
+    "Entrainment factor"
+    c_ε::FT = 0.13
+    "Detrainment factor"
+    c_δ::FT = 0.52
+    "Trubulent Entrainment factor"
+    c_t::FT = 0.1
+    "Detrainment RH power"
+    β::FT = 2
+    "Logistic function scale ‵[1/s]‵"
+    μ_0::FT = 0.0004
+    "Updraft mixing fraction"
+    χ::FT = 0.25
+    "Minimum updraft velocity"
+    w_min::FT = 0.1
+    "Exponential area limiter scale"
+    lim_ϵ::FT = 0.0001
+    "Exponential area limiter amplitude"
+    lim_amp::FT = 10
+    "Minimum Updraft area"
+    a_min::FT = 0.001
+    "Minimum value for turb entr"
+    εt_min::FT = 1e-4
```

Comment:
Should the default parameters here be stored in CLIMAParameters.jl?

Reply:
Yeah, I think we'll need to move some of them to CLIMAParameters. I was a bit hesitant to move them because the model, and maybe the parameters to tune, are still under development. So, I think it might be better to hold off on moving parameters to CLIMAParameters for now, to avoid moving parameters to CLIMAParameters that may eventually be non-existent in the converged version of the EDMF model.

Reply:
some of these parameters hopefully will be avoided in the near future. Charlie suggested we move our parameters only once we have running test cases and we know exactly which parameters we still need